docs/core-features/fhe_basics.md

```diff
@@ -1,45 +1,44 @@
+FHE encrypts data as LWE ciphertexts, represented visually as a bit vector. The encrypted message is located in the higher-order (yellow) bits, while the lower-order (gray) bits contain random noise that ensures the security of the ciphertext.
 
 ![](../\_static/basics/Ciphertext.png)
 
-Under the hood, each time you perform an operation on an encrypted value, the noise grows and at a certain point, it may overlap with the message and corrupt its value.
 
-There is a way to decrease the noise of a ciphertext with the **Bootstrap operation**. The bootstrap operation takes as input a noisy ciphertext and generates a new ciphertext encrypting the same message, but with a lower noise. This allows additional operations to be performed on the encrypted message.
+Each operation on an encrypted value increases the noise, and if it becomes too large, it may overlap with the message and corrupt its value.  To reduce the noise of a ciphertext, the **Bootstrap operation** generates a new ciphertext encrypting the same message, but with lower noise. This allows additional operations to be performed on the encrypted message.
 
-A typical FHE program will be made up of a series of operations followed by a Bootstrap, this is then repeated many times.
+In typical FHE programs, operations are followed by a bootstrap, and this sequence repeats multiple times.
 
 ## Probability of Error
 
 The amount of noise in a ciphertext is not as bounded as it may appear in the above illustration. As the errors are drawn randomly from a Gaussian distribution, they can be of varying size. This means that we need to be careful to ensure the noise terms do not affect the message bits. If the error terms do overflow into the message bits, this can cause an incorrect output (failure) when bootstrapping.
 
-The default failure probability in Concrete is set for the whole program and is $$\frac{1}{100000}$$ by default. This means that 1 execution of every 100,000 may result in an incorrect output. To have a lower probability of error, you need to change the cryptographic parameters, likely resulting in worse performance. On the other side of this trade-off, allowing a higher probability of error will likely speed-up operations.
-
-## Function evaluation
+The noise in a ciphertext isn't strictly bounded, as errors are drawn from a Gaussian distribution and vary in size. If the noise grows too large, it may corrupt the message bits, causing incorrect outputs during bootstrapping.
 
-So far, we only introduced arithmetic operations but a typical program usually also involves functions (maximum, minimum, square root…)
+In Concrete, the default failure probability is set to $$\frac{1}{100000}$$, meaning that 1 in every 100,000 executions may result in an error. Reducing this probability requires adjusting cryptographic parameters, potentially lowering performance. Conversely, allowing a higher probability of error may improve performance.
```

Comment:
@BourgerieQuentin : is 

> $$\frac{1}{100000}$$, meaning that 1 in every 100,000 

still true? I would replace by a constant here, to avoid to say craps.

Reply:
@BourgerieQuentin @umut-sahin

Reply:
It's still the default and I think we should keep it to show what's the default and what they should expect.

Reply:
ok thx